basic_system/src/system_implementation/system/mod.rs

```diff
@@ -132,7 +132,8 @@ impl<R: Resources> StorageAccessPolicy<R, Bytes32> for EthereumLikeStorageAccess
 }
 
 pub type ExtraCheck<SCC: const StackCtorConst, A: Allocator> =
-    [[[[[[[(); SCC::extra_const_param::<(EventContent<MAX_EVENT_TOPICS, A>, ()), A>()];
+    [[[[[[[[(); SCC::extra_const_param::<(u32, ()), A>()];
```

Comment:
Btw, can you pls clarify what this code is about?

Reply:
Oh, I know it comes from the const_trait used for the StackCtorConst trait, but not much more.